app/services/import_matches.rb

```diff
@@ -0,0 +1,82 @@
+class ImportMatches < Patterns::Service
+  class ImportRound < Patterns::Service
+    attr_reader :round
+
+    def initialize(round)
+      @round = round
+    end
+
+    def match_round
+      Round.find_or_create_by(year: round.year, number: round.number)
+    end
+
+    def call
+      UpsertMatches.call(round: match_round, matches: round.matches)
+    end
+  end
+
+  class UpsertMatches < Patterns::Service
+    attr_reader :round, :matches
+
+    def initialize(round:, matches:)
+      @round = round
+      @matches = matches
+    end
+
+    def call
+      matches.each { |match| UpsertMatch.call(match_attributes: match, round: round) }
+    end
+  end
+
+  class UpsertMatch < Patterns::Service
+    attr_reader :round, :match_attributes
+
+    def initialize(round:, match_attributes:)
+      @round = round
+      @match_attributes = match_attributes
+    end
+
+    def call
+      update_football_score
```

Comment:
drop using `football` in names here as we are not differentiating between football and other matches